.gitignore

```diff
@@ -1,3 +1,6 @@
 /coverage
 /node_modules
 /yarn-error.log
+
+# IDEs
+/.idea
```

Comment:
This belongs to a [global gitignore](https://help.github.com/articles/ignoring-files/#create-a-global-gitignore).

Reply:
TIL. Thanks!